pgmpy/models/BayesianNetwork.py

```diff
@@ -594,6 +594,9 @@ def predict(self, data, stochastic=False, n_jobs=-1):
         data: pandas DataFrame object
             A DataFrame object with column names same as the variables in the model.
 
+        algo: pgmpy Inference instance
```

Comment:
A more informative description would be "a subclass of pgmpy.inference.Inference"